canonical-web-and-design/jaas-dashboard
# Add filter panel to input

## Done

Add filter panel to input

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Click the filter input and verify the filter panel appears
- Click anywhere else on the page and verify the disappears
